storages/backends/azure_storage.py

```diff
@@ -98,10 +101,25 @@ def _save(self, name, content):
 
     def url(self, name):
         if hasattr(self.connection, 'make_blob_url'):
+            if self.auto_sign:
+                access_policy = AccessPolicy()
+                access_policy.start = (datetime.utcnow() + timedelta(seconds=-120)).strftime('%Y-%m-%dT%H:%M:%SZ')
```

Comment:
I think we should just leave start to its default since that is when the service receives the blob which seems reasonable to me.